templates/info.hpp.in

```diff
@@ -25,6 +25,8 @@
 #include <array>
 #include <iosfwd>
 #include <string>
+#include <utility>
+#include "boost/optional.hpp"
```

Comment:
Should we not write `<boost/optional.hpp>`?

Reply:
Done.